corecomponent/androidcomponent/src/main/res/navigation/navigation.xml

```diff
@@ -112,6 +112,10 @@
             android:name="speakerId"
             app:argType="io.github.droidkaigi.confsched2020.model.SpeakerId"
             />
+        <argument
+            android:name="transitionNameSuffix"
```

Comment:
I want to distinguish "SessionItem to SpeakerFragment" and "SpeakerItem to SpeakerFragment" at SearchFragment and to avoid setting same transition name to images.
Example is below

![20200118_225025](https://user-images.githubusercontent.com/32016001/72664784-6a5f3400-3a45-11ea-8b8f-ad124f50bf99.gif)

Reply:
🆒